com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -486,6 +500,10 @@ public static void RunPlacement()
                         cellPositions.Add(new Vector3Int(x, y, z));
 
             int index = 0;
+            // For now we just have one baking batch. Later we'll have more than one for a set of scenes.
+            // All probes need to be baked only once for the whole batch and not once per cell
+            // The reason is that the baker is not deterministic so the same probe position baked in two different cells may have different values causing seams artefacts.
+            m_BakingBatch = new BakingBatch(0);
```

Comment:
Even tho for now we have a single batch, I'd avoid passing an hardcoded 0 here, maybe have a static counter to grab next bakingBatch ID? (so that when we split that up we don't need to track this 0 down)

Reply:
Can do that yes

Reply:
Done in latest commit.